samples/cpp/benchmark_app/README.md

```diff
@@ -125,11 +125,12 @@ Options:
                                 estimations the number of streams should be set to 1.
     -nthreads "<integer>"       Optional. Number of threads to use for inference on the CPU (including HETERO and MULTI cases).
     -enforcebf16="<true/false>" Optional. By default floating point operations execution in bfloat16 precision are enforced if supported by platform.
```

Comment:
As far as I see handling of this parameter is removed. Maybe remove it as well?

Reply:
done